packages/graphql-live-query/package.json

```diff
@@ -27,8 +27,8 @@
   "main": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
   "exports": {
-    "require": "dist/index.cjs.js",
-    "default": "dist/index.mjs"
+    "require": "./dist/index.cjs.js",
```

Comment:
Node 12 and VSCode IDE complained about this not being prefixed (could not resolve module), however, bob complained about this being prefixed. I commented the lines out for now. Gotta check whether this is an issue that should be fixed in bob.